T/TempestRemap/build_tarballs.jl

```diff
@@ -73,7 +73,7 @@ products = [
 
 dependencies = [
     Dependency("OpenBLAS32_jll"),
-    Dependency("NetCDF_jll"),
+    Dependency("NetCDF_jll", compat="^400.801.100"),
```

Comment:
400.801.100 [was yanked](https://github.com/JuliaRegistries/General/pull/46656), so lets use 400.801.101:
```suggestion
    Dependency("NetCDF_jll", compat="400.801.101"),
```

Reply:
so do i need the caret or not?

Reply:
The caret is the default: https://pkgdocs.julialang.org/v1/compatibility/#Version-specifier-format